app/controllers/reports/regions_controller.rb

```diff
@@ -118,20 +118,7 @@ def download
   end
 
   def monthly_district_data_report
-    # re-implementing part of the find_region method with a modification as a temporary workaround for this bug:
-    # https://app.clubhouse.io/simpledotorg/story/3380/facilitydistrict-should-be-initialized-with-name-not-slug
-    report_scope = report_params[:report_scope]
-    @region ||= authorize {
-      case report_scope
-      when "facility_district"
-        scope = current_admin.accessible_facilities(:view_reports)
-        FacilityDistrict.new(name: report_params[:id], scope: scope)
-      when "district"
-        current_admin.accessible_district_regions(:view_reports).find_by!(slug: report_params[:id])
-      else
-        raise ActiveRecord::RecordNotFound, "unknown report_scope #{report_scope}"
-      end
-    }
```

Comment:
With this removal, can we also stop specifying a `report_scope` parameter somewhere?

Reply:
We still need to pass `report_scope` everywhere for differentiating between `district`, `facility` and `blocks`.

Reply:
> With this removal, can we also stop specifying a `report_scope` parameter somewhere?

We could get away from using `report_scope`, but it would mean changing our URLs to use Region slug instead of source slug...as the Region slug is truly unique, and then we could do a plain old `Region.find_by(slug: params[:slug])`.  That would definitely be a bigger change, and would break any old URLs that admins may have bookmarked where the Region slug does not match the source slug.